lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -356,6 +387,16 @@ def __init__(self, path, *, zs=(), zdir='z', **kwargs):
         self.set_3d_properties(path, zs, zdir)
 
     def set_3d_properties(self, path, zs=0, zdir='z'):
+        """
+        Set the *z* position and direction of the path patch.
+
+        Parameters
+        ----------
+        z : float
```

Comment:
```suggestion
        zs : float
```
Also, it's allowed to be an array that matches `path`, I think?